internal/service/glue/job.go

```diff
@@ -132,9 +132,10 @@ func ResourceJob() *schema.Resource {
 			"tags":     tftags.TagsSchema(),
 			"tags_all": tftags.TagsSchemaComputed(),
 			"timeout": {
-				Type:     schema.TypeInt,
-				Optional: true,
-				Default:  2880,
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.IntAtLeast(1),
```

Comment:
Does the API have an upper bounds?

Reply:
Not per the documentation, and testing high values such as 300000 work as expected